src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -54,6 +59,22 @@ public class DevBoxInstance : IComputeSystem
 
     private const string DevBoxMultipleConcurrentOperationsNotSupportedKey = "DevBox_MultipleConcurrentOperationsNotSupport";
 
+    private static readonly CompositeFormat ProtocolPinString = CompositeFormat.Parse("ms-cloudpc:pin?location={0}&request={1}&cpcid={2}&workspaceName={3}&environment={4}&username={5}&version=0.0&source=DevHome");
+
+    // this is the version of the Windows App package that supports protocol associations for pinning
+    private static readonly PackageVersion MinimumWindowsAppVersion = new(1, 3, 243, 0);
+
+    // These exit codes must be kept in sync with WindowsApp

```

Comment:
Where is WindowsApp?

Reply:
The code lives in a different ADO git repo.  The app is delivered via the store.